gameOfSticks.py

```diff
@@ -1,6 +1,3 @@
-#! /usr/bin/env python3
-
-from __future__ import print_function
```

Comment:
Why is this removed? :o

Reply:
For me it was of no use, so I just removed while copying the code

Reply:
7.69

Reply:
it showed me where i lost points XD